tldw_Server_API/app/core/DB_Management/chacha/runtime.py

```
@@ -0,0 +1,606 @@
+    start: float,
+) -> None:
+    db_instance: CharactersRAGDB | None = None
+    error: Exception | None = None
+    success = False
+    should_close = False
+    try:
+        db_instance = completed_future.result()
+        success = True
+    except asyncio.CancelledError:
+        return
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)